library/modules/bigip_policy_rule.py

```diff
@@ -84,6 +86,15 @@
       event:
         description:
           - Events on which actions such as reset can be triggered.
+          - With set_variable action, it is used for specifying action event such as request or response.
+        type: str
+      expression:
+        description:
+          - A tcl expression used with set_variable action.
+        type: str
+      tmName:
+        description:
+           - Variable name used with set_variable action.
```

Comment:
change this to something user friendly, do not use camel case parameters. since this is a nested  parameter you need to do something like this:

https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_apm_acl.py#L484

Reply:
Rename the filed to `variable_name`
This comment is addressed now.

Regarding second part of this comment:
>  since this is a nested parameter you need to do something like this:
> https://github.com/F5Networks/f5-ansible/blob/devel/library/modules/bigip_apm_acl.py#L484

I am not adding a new field; I am re-using the existent one; just include additional option: 
https://github.com/andreykashcheev/f5-ansible/blob/FMFA-467/library/modules/bigip_policy_rule.py#L483